cloudbuild.yaml

```diff
@@ -19,347 +19,347 @@
+
+  - name: '${_REPOSITORY}/e2e-tests:${_TAG}'
+    id: 'Run E2E Test for JDBC mode with HAPI source'
+    env:
+      - PARQUET_SUBDIR=JDBC_HAPI
+      - FHIR_JSON_SUBDIR=JDBC_HAPI_FHIR_JSON
+      - DOCKER_NETWORK=--use_docker_network
+      - SINK_SERVER=http://sink-server-jdbc:8080
+    waitFor: ['Run Batch Pipeline for JDBC mode with HAPI source', 'Build E2E Image']
+
+  - name: '${_REPOSITORY}/batch-pipeline:${_TAG}'
+    id: 'Run Batch Pipeline for BULK_EXPORT mode with HAPI source'
+    env:
+      - FHIR_FETCH_MODE=BULK_EXPORT
+      - FHIR_SERVER_URL=http://hapi-server:8080/fhir
+      - PARQUET_PATH=/workspace/e2e-tests/BULK_EXPORT
+      - OUTPUT_PARQUET_VIEW_PATH=/workspace/e2e-tests/BULK_EXPORT/VIEWS_TIMESTAMP_1
+    waitFor: ['Run E2E Test for JDBC mode with HAPI source']
+
+  - name: '${_REPOSITORY}/e2e-tests:${_TAG}'
+    id: 'Run E2E Test for BULK_EXPORT mode with HAPI source'
+    env:
+      - PARQUET_SUBDIR=BULK_EXPORT
+      - FHIR_JSON_SUBDIR=BULK_EXPORT_FHIR_JSON
+      - DOCKER_NETWORK=--use_docker_network
+    waitFor: ['Run Batch Pipeline for BULK_EXPORT mode with HAPI source']
+
+  # Resetting FHIR sink server
+  - name: 'docker/compose'
+    id: 'Turn down FHIR Sink Server Search'
+    args: [ '-f', './docker/sink-compose.yml', '-p', 'sink-server-search', 'down' ,'-v']
+    env:
+      - SINK_SERVER_NAME=sink-server-search
+      - SINK_SERVER_PORT=9001
+    waitFor: ['Run E2E Test for FHIR-search mode with HAPI source']
+
+  - name: 'docker/compose'
+    id: 'Turn down FHIR Sink Server JDBC'
+    args: [ '-f', './docker/sink-compose.yml', '-p', 'sink-server-jdbc', 'down' ,'-v']
+    env:
+      - SINK_SERVER_NAME=sink-server-jdbc
+      - SINK_SERVER_PORT=9002
+    waitFor: ['Run E2E Test for JDBC mode with HAPI source']
+
+  # The `views` database is used for creating flat views from ViewDefinitions.
+  - name: 'postgres'
+    id: 'Create views database'
+    entrypoint: psql
+    env:
+      - PGPASSWORD=admin
+    args: [ '-U', 'admin', '-d', 'postgres', '-h', 'hapi-fhir-db', '-p', '5432',
+            '-c', 'CREATE DATABASE views;']
+    waitFor: ['Turn down FHIR Sink Server Search']
+
+  - name: 'docker/compose'
+    id: 'Bring up controller and Spark containers'
+    env:
+      - PIPELINE_CONFIG=/workspace/docker/config
+      - DWH_ROOT=/workspace/e2e-tests/controller-spark/dwh
+    args: [ '-f', './docker/compose-controller-spark-sql-single.yaml', 'up',
+            '--force-recreate', '-d' ]
+    waitFor: ['Create views database']
+
+  - name: '${_REPOSITORY}/e2e-tests/controller-spark:${_TAG}'
+    id: 'Run E2E Test for Dockerized Controller and Spark Thriftserver'
+    waitFor: [ 'Bring up controller and Spark containers',
+               'Run E2E Test for BULK_EXPORT mode with HAPI source' ]
+
+  # The controller logs don't appear in Cloud Build output because we run it in
+  # the detached mode. For debugging controller failures we can use something like
```

Comment:
Please update this comment with the `allowFailure: true` trick that you used.

Reply:
Added